setup.py

```diff
@@ -29,6 +29,7 @@
         "requests",
         "robotframework-requests",
         "escapism",
+        "semver>=2,<3"
```

Comment:
is there a reason for version b/w 2 and 3?

Reply:
v3 has breaking changes but I think it's still not released, my pip install got me version 2 by default